app/views/trainees/lead_schools/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="govuk-heading-l">Lead school</h1>
+
+<%= register_form_with model: @trainee_lead_school_form, url: trainee_lead_schools_path(@trainee), local: true, html: { data: { module: "app-schools-autocomplete" } } do |f| %>
+  <div class="govuk-form-group">
+    <span class="govuk-hint">Search for a school by it's unique reference number (URN), name or postcode</span>
```

Comment:
As above, this string should be in `en.yml`

Reply:
moved